assert.throws(TypeError, function() {
    Number.MAX_VALUE = 42;
});